ns1/datadog_checks/ns1/data/conf.yaml.example

```diff
@@ -118,7 +118,7 @@ instances:
     ## This changes the collection interval of the check. For more information, see:
     ## https://docs.datadoghq.com/developers/write_agent_check/#collection-interval
     #
-    # min_collection_interval: 15
+    # min_collection_interval: 60
```

Comment:
This default does not get applied unless the user enables this option to override the default `15`. Would it make sense to have this uncommented/enabled by default for new customers? Existing users won't be affected because their configuration files are persisted on integration upgrade.

Reply:
Good suggestion, will do that, thanks